### PR TITLE
Updated Homework Week-1-Fatih 

### PR DESCRIPTION
**Pull request by FatihUcars**
Thanks Yash for the reviews. I updated my homework accordingly (Found some other small mistakes and fixed them too).

I am sorry for being late for the update.

### CODE REVIEW

**Comment by yash-kapila** [comment]
Ah. You need not create a new pull request whenever you update homework based on the feedback. You can simply checkout your week's branch and add your commits there. Once you push that branch again, your existing pull request will get updated. 🙂

File: Week1/homework/js-exercises/ArrayAnimals.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"use strict";
+let vehicles = [];
+console.log('These are some vehicles.');
+console.log(vehicles);
+let favAnimals = ['Cat', 'Dog', 'Elephant'];
```
Comment (yash-kapila) [inline Week1/homework/js-exercises/ArrayAnimals.js:5]: You can still use `const` here as you are not reassigning these variables. 🙂

File: Week1/homework/js-exercises/LengthString.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+"use strict";
+const mySentence = 'Programming is so interesting!';
+let sentenceLength = mySentence.length;
```
Comment (yash-kapila) [inline Week1/homework/js-exercises/LengthString.js:3]: This can be `const` as well

File: Week1/homework/js-exercises/RoundNumber.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+"use strict";
+const z = 7.25;
+console.log(z);
+let a = (Math.round(z));
```
Comment (yash-kapila) [inline Week1/homework/js-exercises/RoundNumber.js:4]: Can be `const` here as well

**Review by yash-kapila — APPROVED** [review]
Hi @FatihUcars - I have reviewed your homework and am approving the same. Good luck!

**Comment by FatihUcars** [comment]
> Ah. You need not create a new pull request whenever you update homework based on the feedback. You can simply checkout your week's branch and add your commits there. Once you push that branch again, your existing pull request will get updated. 🙂

Ah okay. I knew I was doing something wrong :) Thanks!

**Pull request closed without merging**